res/css/views/rooms/_RoomTile.scss

```diff
@@ -153,12 +153,9 @@ limitations under the License.
     background-color: $warning-color;
 }
 
-.mx_RoomTile_unread, .mx_RoomTile_highlight {
-    .mx_RoomTile_name {
-        // Parent weighting won't apply to this because the above mx_RoomTile_name overrides.
-        font-weight: 700;
-        color: $roomtile-selected-color;
-    }
+.mx_RoomTile_unread, .mx_RoomTile_highlight .mx_RoomTile_name {
```

Comment:
these are not equivalent, what was there before will expand to:
```
.mx_RoomTile_unread .mx_RoomTile_name, .mx_RoomTile_highlight .mx_RoomTile_name {
...
}
```
Probably best to keep it as it was to avoid repetition.

Reply:
oh,...yes. Good catch.